### PR TITLE
Fix/abstract styling

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

File: src/lib/components/InputWithIcon.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const InputWithIcon = ({
       onClick={action ? () => AmpTrack(AmpEvent.USE_SEARCH_INPUT) : undefined}
     />
     <InputRightElement h="56px" alignItems="center" mr="1">
```
Comment (songwongtp) [inline src/lib/components/InputWithIcon.tsx:32]: mr={1}

File: src/lib/components/ListSelection.tsx [hunk 2]
```diff
@@ -194,7 +194,7 @@ export const ListSelection = forwardRef<HTMLInputElement, ListSelectionProps>(
           {displayOptions && (
             <List
               borderRadius="8px"
-              bg="pebble.800"
+              bg="gray.800"
               px="2"
               py="1"
```
Comment (songwongtp) [inline src/lib/components/ListSelection.tsx:199]: px={2}
py={1}

File: src/lib/components/ListSelection.tsx [hunk 3]
```diff
@@ -221,7 +221,7 @@ export const ListSelection = forwardRef<HTMLInputElement, ListSelectionProps>(
                         data-label={option.label}
                         mr={2}
                         name="check"
-                        color="pebble.600"
+                        color="gray.600"
                         boxSize="3"
```
Comment (songwongtp) [inline src/lib/components/ListSelection.tsx:225]: boxSize={3}

File: src/lib/components/TagSelection.tsx [hunk 4]
```diff
@@ -201,7 +201,7 @@ export const TagSelection = observer(
             {displayOptions && (
               <List
                 borderRadius="8px"
-                bg="pebble.800"
+                bg="gray.800"
                 px="2"
                 py="1"
```
Comment (songwongtp) [inline src/lib/components/TagSelection.tsx:206]: ditto

File: src/lib/components/TxFilterSelection.tsx [hunk 5]
```diff
@@ -178,7 +178,7 @@ export const TxFilterSelection = forwardRef<
           {displayOptions && (
             <List
               borderRadius="8px"
-              bg="pebble.900"
+              bg="gray.900"
               px="2"
               py="1"
```
Comment (songwongtp) [inline src/lib/components/TxFilterSelection.tsx:183]: ditto

File: src/lib/components/button/CustomIconButton.tsx [hunk 6]
```diff
@@ -30,7 +30,7 @@ export const CustomIconButton = ({
       alignItems="center"
       onClick={onClick}
     >
-      <CustomIcon name={icon} boxSize="5" color="pebble.600" />
+      <CustomIcon name={icon} boxSize="5" color="gray.600" />
```
Comment (songwongtp) [inline src/lib/components/button/CustomIconButton.tsx:33]: boxSize={5}

File: src/lib/components/button/BackButton.tsx [hunk 7]
```diff
@@ -10,7 +10,7 @@ export const BackButton = (props: ButtonProps) => {
 
   return (
     <Button
-      variant="ghost-lilac"
+      variant="ghost-secondary"
       size="sm"
       mb="8px"
```
Comment (songwongtp) [inline src/lib/components/button/BackButton.tsx:15]: mb={2}

File: src/lib/components/modal/NoMobile.tsx [hunk 8]
```diff
@@ -22,7 +22,7 @@ export const NoMobile = () => (
       >
         <CustomIcon
           name="info-circle-solid"
-          color="violet.light"
+          color="primary.light"
           boxSize="10"
```
Comment (songwongtp) [inline src/lib/components/modal/NoMobile.tsx:26]: boxSize={10}

File: src/lib/components/select-code/CodeSelectDrawerButton.tsx [hunk 9]
```diff
@@ -88,8 +88,8 @@ export const CodeSelectDrawerButton = ({
       <Drawer isOpen={isOpen} onClose={onClose} placement="bottom">
         <DrawerOverlay />
         <DrawerContent h="80%">
-          <DrawerHeader borderBottom="1px solid" borderColor="pebble.700">
-            <CustomIcon name="code" boxSize="6" color="pebble.600" />
+          <DrawerHeader borderBottom="1px solid" borderColor="gray.700">
+            <CustomIcon name="code" boxSize="6" color="gray.600" />
```
Comment (songwongtp) [inline src/lib/components/select-code/CodeSelectDrawerButton.tsx:92]: boxSize={6}

File: src/lib/components/select-contract/SelectContractAdmin.tsx [hunk 10]
```diff
@@ -81,7 +81,7 @@ export const SelectContractAdmin = ({
             <CustomIcon
               name="contract-address-solid"
               boxSize="5"
```
Comment (songwongtp) [inline src/lib/components/select-contract/SelectContractAdmin.tsx:83]: boxSize={5}

File: src/lib/components/select-contract/SelectContractInstantiator.tsx [hunk 11]
```diff
@@ -124,7 +124,7 @@ export const SelectContractInstantiator = ({
                 <CustomIcon
                   name="contract-address-solid"
                   boxSize="5"
```
Comment (songwongtp) [inline src/lib/components/select-contract/SelectContractInstantiator.tsx:126]: ditto

File: src/lib/components/table/contracts/EditTags.tsx [hunk 12]
```diff
@@ -31,20 +31,15 @@ export function EditTags({ contractLocalInfo }: EditTagsProps) {
     <ActionModal
       title="Edit Tags"
       trigger={
-        <CustomIcon
-          cursor="pointer"
-          name="edit"
-          color="pebble.600"
-          boxSize="3"
-        />
+        <CustomIcon cursor="pointer" name="edit" color="gray.600" boxSize="3" />
```
Comment (songwongtp) [inline src/lib/components/table/contracts/EditTags.tsx:34]: boxSize={3}

File: src/lib/pages/query/index.tsx [hunk 13]
```diff
@@ -100,7 +100,7 @@ const Query = () => {
         </Heading>
         <Box>
           <Button
-            variant="ghost-lilac"
+            variant="ghost-secondary"
             size="sm"
             p="unset"
             pl="2"
```
Comment (songwongtp) [inline src/lib/pages/query/index.tsx:106]: pl={2}

**Review by songwongtp — APPROVED** [review]

File: src/lib/components/button/BackButton.tsx (hunk 7 above)
Comment (bkioshn) [reply src/lib/components/button/BackButton.tsx:15]: pr={2}

File: src/lib/pages/tx-details/components/TxHeader.tsx [hunk 14]
```diff
@@ -13,7 +13,7 @@ interface TxHeaderProps extends FlexProps {
 }
 
 const DotSeparator = () => (
-  <Box bg="lilac.darker" borderRadius="50%" w="4px" h="4px" />
+  <Box bg="secondary.darker" borderRadius="50%" w="4px" h="4px" />
```
Comment (bkioshn) [inline src/lib/pages/tx-details/components/TxHeader.tsx:16]: w={1} h={1}

File: src/lib/pages/network-overview/index.tsx [hunk 15]
```diff
@@ -64,11 +64,7 @@ const CardInfo = ({
         </Text>
         <Tooltip label={tooltip}>
           <Flex cursor="pointer">
-            <CustomIcon
-              name="info-circle-solid"
-              boxSize="3"
-              color="pebble.600"
-            />
+            <CustomIcon name="info-circle-solid" boxSize="3" color="gray.600" />
```
Comment (bkioshn) [inline src/lib/pages/network-overview/index.tsx:67]: {3}

File: src/lib/pages/contract-list/slug.tsx [hunk 16]
```diff
@@ -84,7 +84,7 @@ const ContractsByList = observer(() => {
         w="full"
         spacing="4px"
         separator={
-          <CustomIcon name="chevron-right" boxSize="3" color="pebble.600" />
+          <CustomIcon name="chevron-right" boxSize="3" color="gray.600" />
```
Comment (bkioshn) [inline src/lib/pages/contract-list/slug.tsx:87]: {3}

File: src/lib/components/DotSeperator.tsx [hunk 17]
```diff
@@ -1,5 +1,5 @@
 import { Box } from "@chakra-ui/react";
 
 export const DotSeparator = () => (
-  <Box bg="lilac.darker" borderRadius="50%" w="4px" h="4px" />
+  <Box bg="secondary.darker" borderRadius="50%" w="4px" h="4px" />
```
Comment (bkioshn) [inline src/lib/components/DotSeperator.tsx:4]: Remove this file

File: src/lib/components/state/ZeroState.tsx [hunk 18]
```diff
@@ -22,7 +22,7 @@ interface ActionSectionProps {
 const ActionSection = ({ list, handleAction }: ActionSectionProps) =>
   list.value === formatSlugName(INSTANTIATED_LIST_NAME) ? (
     <Button
-      leftIcon={<CustomIcon name="add-new" boxSize="4" color="text.main" />}
+      leftIcon={<CustomIcon name="add-new" boxSize="4" />}
```
Comment (bkioshn) [inline src/lib/components/state/ZeroState.tsx:25]: {4}

File: src/lib/components/modal/RedoModal.tsx [hunk 19]
```diff
@@ -44,13 +44,13 @@ export const RedoModal = ({ message }: RedoModalProps) => {
         <ModalContent w="640px">
           <ModalHeader>
             <Flex w="full" direction="row" alignItems="center" gap={2} pt={1}>
-              <CustomIcon name="redo" boxSize="5" color="pebble.600" />
+              <CustomIcon name="redo" boxSize="5" color="gray.600" />
```
Comment (bkioshn) [inline src/lib/components/modal/RedoModal.tsx:47]: {5}

**Review by evilpeach — APPROVED** [review]

**Pull request merged**